lib/client/Client.js

```diff
@@ -110,33 +156,34 @@ export default class Client {
    * @param deviceName {String} A unique device name
    * @param continuationCallback {Function} A function that takes an {@link AuthContinuation} object as argument, and should return a promise, that will resolve with the continuation token.
    *
-   * @return {Client} This Client instance.
+   * @return {Promise} A {@link Promise} that will eventually be resovled with a {@link AuthAccess} object
    */
-
   authExternal(username, deviceName, continuationCallback) {
```

Comment:
You can remove this continuationCallback, the only callback needed by `authenticate` is provided by this method.